### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 - es. 3.13 (28 ottobre)

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/calcolo-delle-probabilita#303] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/8aa8bfd4-3f7d-4a03-a981-a2b15791794c)

**Comment by SickRickX2** [comment]
![image](https://github.com/user-attachments/assets/14cfa25b-dd82-4f99-a320-61fd1fe6a8f7)

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/ff1ef4b1-593a-48db-afc9-4bc3c77c40ef)

**Reply by rimaout** [reply]
<img width="400" alt="image" src="https://github.com/user-attachments/assets/0bb2711c-fbd6-44df-877a-c34f5a51920f">
